locale/fi/LC_MESSAGES/django.po

```diff
@@ -1014,48 +1015,22 @@ msgid ""
 "<dl>\n"
 "\t<dt>XXXXX1 and XXXXX2</dt>\n"
 "    <dd>the real student ids</dd>\n"
-"\t<dt>X1 (integer)</dt>\n"
-"    <dd>the system userprofile id of the desired grader</dd>\n"
+"\t<dt>X1 (integer)</dt> <p>Your userprofile id is <b>%(id)s</b>.</p>\n"
 "\t<dt>Z1 and Z2 (integers)</dt>\n"
 "    <dd>the system exercise ids</dd>\n"
 "</dl>\n"
-"<pre>{\n"
```

Comment:
Was this example JSON remvoed on purpose (why?) or is the diff just looking odd?

Reply:
Translation file